calico_containers/pycalico/ipam.py

```diff
@@ -33,7 +33,7 @@
 
 _log = logging.getLogger(__name__)
 
-IPAM_V_PATH = "/calico/ipam/v1/"
+IPAM_V_PATH = "/calico/ipam/v2/"
```

Comment:
Have we estimated how much work it would be to make the IPAM code resilient to different block sizes?

If it isn't an excessive amount, it might save us down the road to support existing blocks, and any new blocks will be 64 addresses in size.